pulp/apis/coin_api.py

```diff
@@ -167,6 +170,8 @@ def solve_CBC(self, lp, use_mps=True):
             cmds += f"mips {tmpMst} "
         if self.timeLimit is not None:
             cmds += f"sec {self.timeLimit} "
+        if self.maxNodes is not None:
```

Comment:
As the other comment says, this attribute does not exist. You should thus do something like:
```
maxNodes = self.optionsDict.get("maxNodes")
if maxNodes is not None:
    cmds += f"maxNodes {maxNodes} "
```